[Question] How to specify JAVA_VERSION

Not sure how to specify that I'm using JAVA 11.
It's taking by default 8

```
$ lsif-java index
Using JVM version '8'
$ /builds/springboot-sandbox/gradlew --init-script /tmp/lsif-java7756353260882540485/java-toolchains.gradle lsifDetectJavaToolchains
Downloading https://services.gradle.org/distributions/gradle-6.8.3-bin.zip
..........10%..........20%..........30%...........40%..........50%..........60%..........70%...........80%..........90%..........100%
Welcome to Gradle 6.8.3!
Here are the highlights of this release:
 - Faster Kotlin DSL script compilation
 - Vendor selection for Java toolchains
 - Convenient execution of tasks in composite builds
 - Consistent dependency resolution
For more details see https://docs.gradle.org/6.8.3/release-notes.html
Starting a Gradle Daemon (subsequent builds will be faster)
> Task :lsifDetectJavaToolchains
BUILD SUCCESSFUL in 24s
1 actionable task: 1 executed
$ /builds/springboot-sandbox/gradlew --no-daemon --init-script /tmp/lsif-java7756353260882540485/init-script.gradle -Porg.gradle.java.installations.auto-detect=false -Porg.gradle.java.installations.auto-download=false -Porg.gradle.java.installations.paths=/tmp/lsif-java7756353260882540485/1.11 clean compileTestJava lsifJavaDependencies
To honour the JVM settings for this build a single-use Daemon process will be forked. See https://docs.gradle.org/6.8.3/userguide/gradle_daemon.html#sec:disabling_the_daemon.
Daemon will be stopped at the end of the build 
> Task :clean
> Task :compileJava FAILED
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':compileJava'.
> The new Java toolchain feature cannot be used at the project level in combination with source and/or target compatibility
* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
* Get more help at https://help.gradle.org
BUILD FAILED in 5s
```

Why it cannot detect that this project is using java 11?
I'm guessing a wrong setting in my side?

I've added  in build.gradle
```
java {
    toolchain {
        languageVersion = JavaLanguageVersion.of(11)
    }
}
```
without success

However, If I remove 
```
sourceCompatibility = JavaVersion.VERSION_11
targetCompatibility = JavaVersion.VERSION_11
```
I don't have the issue above but it is not correct as it is using java8,  my project is using java-11.

--- a/docs/getting-started.md
+++ b/docs/getting-started.md
@@ -28,9 +28,29 @@ $ src lsif upload # (optional) upload index to Sourcegraph
 If everything went OK, a `dump.lsif` file should exist after the command has
 finished indexing the project.
 
-> The Docker container re-downloads all dependencies on every run, which can be
-> slow for large projects. Consider using the [Java launcher](#java-launcher)
-> instead to re-use the existing build cache of your project.
+> The Docker container is made available for convenience at the cost of
+> performance. Consider using the [Java launcher](#java-launcher) instead of the
+> Docker container for better performance. The Docker container is a large
+> download because it includes multiple pre-installed Java versions (Java 8,
+> Java 11 and Java 17). Also, external dependencies of your codebase get
+> re-downloaded on every `lsif-java index` invocation.
+
+Java 8 is the Java version in the Docker container. Use the following commands
+to change the default version:
+
+- Java 11: `eval "$(coursier java --jvm 11 --env)"`
+- Java 17:
+  `eval "$(coursier java --jvm temurin:17 --jvm-index https://github.com/coursier/jvm-index/blob/master/index.json --env)"`
+
+Alternatively, create a `lsif-java.json` file at the root of your repository
+with the following values:
+
+```jsonc
+// lsif-java.json
+{
+  "jvm": "11" // or "17"
+}
+```
 
 ### Java launcher
 

